### DISCUSSION TITLE
Using Symlinks with PhotoPrism

### DISCUSSION THREAD

**Discussion by adamzvolanek** [photoprism/photoprism#1266] [Q&A] [answered]
To give a run-down, I believe I have found a bug in Photoprism in its inability to follow through symlinks within its storage path directory.

My pictures are organized as follows:

> /mnt/user/Pictures/PhotoAlbum/Year/Location -> ../../Personal/Year/Location/*.jpg

> Symlink is named Location so on windows it appears as ...\Pictures\PhotoAlbum\2007\Colorado\*.jpg

If I place a pictures within the /mnt/user/Pictures/PhotoAlbum/Year/Location/*.jpg, it reads in the pictures on the import. However the same pictures location behind the symlink do not get logged.

**Comment by lastzero** [comment, marked as answer]
Are you using Docker? You need to mount the folders. Multiple folders can be mounted as shown in our docs.

Containers are technically not able to see external folders that were not mounted. That's why using them is (more) secure. Thus it's not possible to follow symlinks. In addition, the relative and absolute paths inside and outside containers are typically different.

**Reply by tomasdev** [reply]
So, if symlink source is mounted... it should work even in Docker?

**Reply by tomasdev** [reply]
To answer myself: yes.

1) Mount originals to the folder that contains symlinks
2) Mount the symlinks as additional storage, ensuring the mounted path is exactly the same as in the host

It just works.
